### PR TITLE
WAVE2-650 bulk update fix

### PR DESCRIPTION
**Pull request by balrajb**

### CODE REVIEW

File: sites/all/modules/ucalgary/content_ownership/content_ownership_bulk/content_ownership_bulk.module [hunk 1]
```diff
@@ -817,4 +824,54 @@ function content_ownership_bulk_views_api() {
         'api' => 3.0,
         'template path' => $path . '/templates',
     );
+}
+
+/**
+ * Implements hook_block_info().
+ * This is the block that displays the ownership groups user belongs to,
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_bulk/content_ownership_bulk.module]: Should be made a bit clearer. Are we talking about the current user?

File: sites/all/modules/ucalgary/content_ownership/content_ownership_bulk/content_ownership_bulk.module [hunk 2]
```diff
@@ -817,4 +824,54 @@ function content_ownership_bulk_views_api() {
...
+                $block['content'] = get_ownership_group_filter();
+                break;
+            }
+    }
+    return $block;
+}
+
+/**
+ * Helper function to get a list of groups author belongs to,
+ * so they can click on the group name and filter the dashboard contents accordingly.
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_bulk/content_ownership_bulk.module]: "contents" should be singular "content"

File: sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc [hunk 3]
```diff
@@ -41,7 +41,7 @@ function content_ownership_form_ctools_export_ui_list_form_alter(&$form, &$form_
  * We need a alter the main /admin/structure/views/add page.
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: Should be changed to "We need to alter"

Comment (balrajb) [inline sites/all/modules/ucalgary/content_ownership/content_ownership_core/views_ownership.inc]: @dpfitzsi requested changes committed, thanks for review

**Pull request merged**